.github/workflows/check.yml

```diff
@@ -7,11 +7,19 @@ on:
       - main
   # ..and any pull request.
   pull_request:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: 'The branch, tag or SHA to checkout (uses the default branch if empty)'
```

Comment:
Sorry, I got confused, we can actually remove this input 🤦 
When we select the workflow branch when triggering manually, it checks out the full branch. No need to have the additional param.

Also to test manually before merging you can use GH cli:
```
gh workflow run "Check" --ref tomm/checks-schedule-manual
```

<img width="341" alt="Screenshot 2022-08-26 at 16 27 37" src="https://user-images.githubusercontent.com/57791/186927011-7fe2d46d-12b5-4c9b-98e0-4e1d4e3979ef.png">